packages/kms-local/src/key-management-system.ts

```diff
@@ -257,14 +264,19 @@ export class KeyManagementSystem extends AbstractKeyManagementSystem {
    * @returns a `0x` prefixed hex string representing the signed digest in compact format
    */
   private eth_rawSign(managedKey: string, data: Uint8Array) {
-    return new SigningKey('0x' + managedKey).signDigest(data).compact
+    // Is compact serialized really what we want?
+    return new SigningKey('0x' + managedKey).sign(data).compactSerialized
   }
 
   /**
    * @returns a base64url encoded signature for the `EdDSA` alg
    */
   private async signEdDSA(key: string, data: Uint8Array): Promise<string> {
-    const signer = EdDSASigner(arrayify(key, { allowMissingPrefix: true }))
+    //TODO: Make sure this works as we removed the option from the arrayify method
+    if (key && key.substring(0, 2) !== "0x") {
+      key = "0x" + key;
+    }
+    const signer = EdDSASigner(getBytes(key))
```

Comment:
It's easier to use `@veramo/utils#hexToBytes`. It will take care of prefixes as well.

```suggestion
    const signer = EdDSASigner(hexToBytes(key))
```

Also true for the next few signing methods in this class